hack/make-rules/Makefile.manifest

```diff
@@ -1,30 +0,0 @@
-# Copyright 2018 The Kubernetes Authors.
```

Comment:
I think it is still been refered in https://github.com/kubernetes/kubernetes/blob/master/test/images/Makefile#L15, need to check whether is it really been used anywhere in the `test/images` or just a stale entry!

Reply:
Its a miss. Thought of deleting, but missed. Will update it. Thank you.